airbyte-metrics/reporter/src/main/resources/application.yml

```diff
@@ -0,0 +1,41 @@
+micronaut:
+  application:
+    name: airbyte-metrics-reporter
+  security:
+    intercept-url-map:
+      - pattern: /**
+        httpMethod: GET
+        access:
+          - isAnonymous()
+  server:
+    port: 9000
```

Comment:
9000 is the port that `airbyte-workers` used before the conversion.  I'm not sure if this will be an issue when running via Docker Compose, as that port is already mapped to something.

Reply:
@jdpgrailsdev looks like we're using the same port in a few places:
   - [cron](https://github.com/airbytehq/airbyte/blob/65a35b98bd42325942989972fed18277a5f29aa7/airbyte-cron/src/main/resources/application.yml#L11)
   - [workers](https://github.com/airbytehq/airbyte/blob/0ace81637e5adfeaaf76ee1ce06fd0641a34c860/airbyte-workers/src/main/resources/application.yml#L11)

Reply:
It was originally added to `airbyte-workers`, as it was already using port 9000.  I believe it is copy/pasta in `airbyte-cron`.  This is probably fine because the containers/pods will handle mapping to it and we can cycle back later if we want to have the services on different ports for identification, etc.